packages/collector/test/tracing/cloud/azure/blob/app.js

```diff
@@ -0,0 +1,328 @@
+/*
+ * (c) Copyright IBM Corp. 2023
+ */
+
+/* eslint-disable no-console */
+
+'use strict';
+
+require('../../../../..')();
+
+const { BlobServiceClient, BlobBatchClient, BlobClient, StorageSharedKeyCredential } = require('@azure/storage-blob');
+
+const express = require('express');
+const bodyParser = require('body-parser');
+const port = require('../../../../test_util/app-port')();
+
+const app = express();
+const logPrefix = `Express / azure blob App (${process.pid}):\t`;
+const fs = require('fs');
+
+const filePath = `${__dirname}/sample.pdf`;
+const localFilePath = `${__dirname}/out.pdf`;
+const data1 = fs.readFileSync(filePath);
+const accountKey = process.env.AZURE_ACCOUNT_KEY;
+const connStr = process.env.AZURE_CONNECTION_STRING;
+const storageAccount = process.env.AZURE_STORAGE_ACCOUNT;
+
+const blobServiceClient = BlobServiceClient.fromConnectionString(connStr);
+const containerName = process.env.AZURE_CONTAINER_NAME;
+const blobName = 'first.pdf';
```

Comment:
As far as I understand, on our storage `teamnodejstracer` there is already a file called `first.pdf`.
If we need to use a new storage, the test would no longer work. Is that correct?

Reply:
Every time the test suite runs, it will create a new container in the storage account and do file operations  specified in tests  in that container and in the end the container is deleted. In testcases we are doing the file upload whenever it requires (not storing any file in prior). The existing file is part of some test runs that might have stopped in the middle. I have deleted that and performed rerun to make sure that testcases still work.

Reply:
Thanks 🙏

Reply:
> As far as I understand, on our storage `teamnodejstracer` there is already a file called `first.pdf`. If we need to use a new storage, the test would no longer work. Is that correct?

We are not keeping any files in prior. The existing file is part of some cancelled test run. I have removed it as tested. For every test run , we will create a container, perform file operations and will delete container at the end of test suite. All file upload operations is done in individual testcases whenever it is required. And for a new storage account, we just need to configure the account name and key in the environment variables